tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -372,22 +363,26 @@ install_kernel_helper() {
 	local kernel_version_yaml_path="${1}"
 	local kernel_name="${2}"
 	local extra_cmd="${3:-}"
+	local extra_tarballs=""
 
 	export kernel_version="$(get_from_kata_deps ${kernel_version_yaml_path})"
 	export kernel_kata_config_version="$(cat ${repo_root_dir}/tools/packaging/kernel/kata_config_version)"
-	local module_dir=""
 
 	if [[ "${kernel_name}" == "kernel-sev" ]]; then
 		kernel_version="$(get_from_kata_deps assets.kernel.sev.version)"
-		default_patches_dir="${repo_root_dir}/tools/packaging/kernel/patches"
-		module_dir="${repo_root_dir}/tools/packaging/kata-deploy/local-build/build/kernel-sev/builddir/kata-linux-${kernel_version#v}-${kernel_kata_config_version}/lib/modules/${kernel_version#v}"
 	elif [[ "${kernel_name}" == "kernel"*"-confidential" ]]; then
 		kernel_version="$(get_from_kata_deps assets.kernel.confidential.version)"
-		default_patches_dir="${repo_root_dir}/tools/packaging/kernel/patches"
-		module_dir="${repo_root_dir}/tools/packaging/kata-deploy/local-build/build/kernel-confidential/builddir/kata-linux-${kernel_version#v}-${kernel_kata_config_version}/lib/modules/${kernel_version#v}"
 	fi
 
-	install_cached_kernel_tarball_component ${kernel_name} ${module_dir} && return 0
+	if [ "${kernel_name}" == "kernel-sev" ] || [ "${kernel_name}" == "kernel"*"-confidential" ]; then
```

Comment:
does this need double braces to get the wildcard matching properly:
```
$ kernel_name=kernel-confidential; [ "${kernel_name}" == "kernel"*"-confidential" ] && echo "matched"
$ kernel_name=kernel-confidential; [[ "${kernel_name}" == "kernel"*"-confidential" ]] && echo "matched"
matched
```

Reply:
Fixed, thanks!